### DISCUSSION TITLE
Add Plugin Support

### DISCUSSION THREAD

**Discussion by Infro** [amitmerchant1990/notepad#36] [Ideas]
Add the ability to add/remove plugins like voice recognition, chatgpt interface, github interface, and etc. that can be configured to be loaded only if used.

**Comment by Infro** [comment]
Can't assign it to myself, but *shrug*

**Comment by amitmerchant1990** [comment]
I want to keep it as simple as possible. And so, I'm not planning to stuff it with a whole lot of features.

We can think about adding voice recognition but that's about it.
